deltachat-ios/Extensions/UIImageView+Extensions.swift

```diff
@@ -0,0 +1,17 @@
+import UIKit
+import DcCore
+
+extension UIImageView {
+
+    func loadVideoThumbnail(from url: URL, placeholderImage: UIImage?, completionHandler: ((UIImage?)->Void)?) {
+
+        self.image = placeholderImage
+        DispatchQueue.global(qos: .background).async {
+            let thumbnailImage = DcUtils.generateThumbnailFromVideo(url: url)
+            DispatchQueue.main.async { [weak self] in
+                self?.image = thumbnailImage
+                completionHandler?(thumbnailImage)
```

Comment:
referring to https://github.com/deltachat/deltachat-ios/pull/787/files#r441420963, instead of the completion handler maybe setNeedsDisplay() is sufficient here?

Reply:
I am using the completion handler here to cache the image, so I would like to keep the completion handler here.

Reply:
I think we will also want to cache the images in other cases when we call this extension method, so we don't need to do it in a completion handler. Anyways, it can be refactored later, too.